File: Exercises/JavaScript/JavaScript_Basics/Functions/ex6.js
 // Write a function that compares the length of two strings. It should return -1 if the first string is shorter, 1 if the second string is shorter, and 0 if they're of equal length, as in the following example:

compareByLength('patience', 'perseverance'); // -1
compareByLength('strength', 'dignity');      //  1
compareByLength('humor', 'grace');           //  0

function compareByLength(str1, str2) {
  if (str1.length < str2.length) {
    console.log("-1");
  } else if (str2.length < str1.length) {
    console.log("1");
  } else {
    console.log("0");
  }
}

File: Exercises/Python/Python_Basics/5_Functions/ex6.py
'''
Write a function that compares the length of two strings. It should return -1 if the first string is shorter, 1 if the second string is shorter, and 0 if they're of equal length. For example:
'''

def compare_by_length(str1, str2):
    len1, len2 = [len(str1), len(str2)]
    if len1 > len2:
        return 1
    elif len1 < len2:
        return -1
    else:
        return 0

print(compare_by_length('patience', 'perseverance')) # -1
print(compare_by_length('strength', 'dignity'))      #  1
print(compare_by_length('humor', 'grace'))           #  0
